### PR TITLE
[naijatype] Update keyboard to tidy code and address issues

### PR DESCRIPTION
**Pull request by LornaSIL**
- v62 was never actually build since the .kmn version wasn't updated.
- Removed unused AndikaAfr.ttf file
- Attempt to address: #3039
- Attempt to address: https://github.com/keymanapp/keyboards/issues/3043

@mcdurdin since I haven't done much with the css on Keyman can you confirm that I've done the right thing?

### CODE REVIEW

File: experimental/n/naijatype/source/naijatype.css
Comment (mcdurdin) [inline experimental/n/naijatype/source/naijatype.css:1]: The `h1` rule also needs to be removed

File: experimental/n/naijatype/source/naijatype.css [hunk 1]
```diff
@@ -3,10 +3,6 @@ font-size: 250%;
 color: rgb(200,10,10);
     }
 
-body {
-    background: rgb(193, 194, 193);
-    font-family: "Andika","DejaVu Sans","AndikaAfr","Andika Afr","Andika New Basic Compact";
-}
 .kmw-key-row {
 }
 /* easier selecting from below *//*
```
Comment (mcdurdin) [inline experimental/n/naijatype/source/naijatype.css:8]: We could probably remove lines 8-14 as they are commented out

**Review by mcdurdin — DISMISSED** [review]
I think looks okay, but some more adjustment to CSS needed

**Review by DavidLRowe — APPROVED** [review]

**Comment by LornaSIL** [comment]
@rowbory We had to make a change to some of the code in your keyboard because the css was making the site look bad. Before you make changes to your version of the keyboard you should make sure you have the up-to-date files from the repo.
Thanks!

**Pull request merged**